test/models/intervention_input_test.rb

```diff
@@ -58,5 +58,30 @@
 
 class InterventionInputTest < ActiveSupport::TestCase
   test_model_actions
+
+  test 'method average_cost_amount' do
+    i = {
```

Comment:
Replace `i = { ... }` directly with `i_o = InterventionInput.new( ... )` and `intervention` is preferred to `i_o`.